lib/rules/no-click-in-non-interactive-elements.js

```diff
@@ -0,0 +1,40 @@
+const NON_INTERACTIVE_ELEMENTS = [
```

Comment:
Tal vez se podría hacer que no salga el error si tiene `role="button"`
estaba viendo https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-element-to-interactive-role.md

Es interesante el approach que usan https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/rules/no-noninteractive-element-interactions.js, sacan la lista de items directamente de la definición de Aria.

Reply:
lo de role button lo pensé, pero no lo usamos mucho así que no le di mucha prioridad. Lo puedo dejar como tarjeta de builder

Sobre lo de Aria, por lo que vi ahí usan la lista de todos los elementos para no analizar elementos custom. La lógica para ver si es elemento interactivo o no es más compleja igual, [está acá](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/util/isInteractiveElement.js). Por lo que veo usa aria también y compara a partir de los roles. Quizás estaría bueno copiarlo? No se que tan necesario sea igual

Reply:
Lo dejaría como mejora futura, que avise cuando se usa `@click` en un div cubre como 99% de nuestros casos de uso.